test/integration/user_visits_place_test.rb

```diff
@@ -1,6 +1,8 @@
 require "test_helper"
 class UserVisitsPlaceSpec < ActionDispatch::IntegrationTest
   include Capybara::DSL
+  require 'vcr'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.